rollup.config.js

```diff
@@ -106,6 +107,7 @@ export default async function ({ watch }) {
           plugins: [
             { resolveFileUrl },
             OMT({ loader: await omtLoaderPromise }),
+            importMetaAssets(),
```

Comment:
Gonna flag this one for discussion. This automatically turns any `new URL("some-file.png", import.meta.url)` into an asset. That means that Rust and Emscripten glue code refer to the correct `.wasm` out of the box and we don’t need to pass it around manually. I’m not sure I like that pattern, as it’s not in line with how the platform handles this without. But I’m gonna yield to @jakearchibald here, it’s his area more than mine.

(As in, if it was up to me, I’d prefer we continue to explicitly import assets with `url:` prefix.)